src/player.js

```diff
@@ -647,6 +647,26 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  get stats(): PKStatsObject {
+    const videoElement = this.getVideoElement();
+    let statsObject: PKStatsObject = {
+      targetBuffer: NaN,
+      availableBuffer: NaN
+    };
+    if (this._engine) {
```

Comment:
You can also substitute this with `if (videoElement)` and remove it from the bottom check as `this.getVideoElement()` validates for `_engine` as well so you can omit it here

Reply:
I know it sounds weird, but can there be an engine but no video elements?

Reply:
No, `this._el` in the engines, which is the response of `getVideoElement` is done in engine constructor.
Also, as mentioned, `this.getVideoElement()` is protected by itself with check fir `_engine`